### PR TITLE
feat(wifi_iot): get info about AP configured for android>=30 [#134]

### PR DESCRIPTION
**Pull request by percymudzinga**

### CODE REVIEW

File: packages/wifi_iot/lib/wifi_iot.dart [hunk 1]
```diff
@@ -5,13 +5,7 @@ import 'dart:io';
 
 import 'package:flutter/services.dart';
 
-enum WIFI_AP_STATE {
```
Comment (Hallot) [inline packages/wifi_iot/lib/wifi_iot.dart:8]: Can we do without the reformatting please?

File: packages/wifi_iot/lib/wifi_iot.dart [hunk 2]
```diff
@@ -379,12 +361,33 @@ class WiFiForIoTPlugin {
     return sResult;
   }
 
+  static Future<String?> getHostpotSSID() async {
```
Comment (daadu) [inline packages/wifi_iot/lib/wifi_iot.dart:364]: Can we not use the `getWiFiAPSSID` API.

File: packages/wifi_iot/lib/wifi_iot.dart [hunk 3]
```diff
@@ -379,12 +361,33 @@ class WiFiForIoTPlugin {
...
+    String? sResult;
+    try {
+      sResult = await _channel.invokeMethod('getHostpotSSID', htArguments);
+    } on MissingPluginException catch (e) {
+      print("MissingPluginException : ${e.toString()}");
+    }
+    return sResult;
+  }
+
+  static Future<String?> getHostpotPassword() async {
```
Comment (daadu) [inline packages/wifi_iot/lib/wifi_iot.dart:375]: Should be renamed to `getWiFiAPPassword` - in line with other AP related APIs.

**Review by daadu — CHANGES_REQUESTED** [review]
Also +1 - if you can remove reformatting changes.

File: packages/wifi_iot/lib/wifi_iot.dart (hunk 3 above)
Comment (daadu) [reply packages/wifi_iot/lib/wifi_iot.dart:375]: Sorry we already have `getWiFiAPPreSharedKey`, can we use the same API instead.

**Comment by daadu** [comment]
@percymudzinga Thank you for working on it.

Related issue for this PR is #134, encourage you to go through it and implement other methods also if possible.

File: packages/wifi_iot/lib/wifi_iot.dart (hunk 1 above)
Comment (daadu) [reply packages/wifi_iot/lib/wifi_iot.dart:8]: @percymudzinga merging https://github.com/percymudzinga/WiFiFlutter/pull/1 should fix this.

Comment (daadu) [reply packages/wifi_iot/lib/wifi_iot.dart]: Looks like I can do it myself. Resolving this. done with https://github.com/florian-guillemard/WiFiFlutter/pull/215/commits/703dad27d09ab2f25be2e353836cf4f66b4975e6

**Comment by daadu** [comment]
Closing this in favour of - #215

**Pull request closed without merging**